app/utils.py

```diff
@@ -549,6 +549,9 @@ class BaseHandler(webapp.RequestHandler):
     # Set this to True to enable a handler even for deactivated repositories.
     ignore_deactivation = False
 
+    # Handlers that don't neet a admin permission can set this False.
```

Comment:
Its default is False, so it would be better to explain when we need to set this to True instead. Also s/neet/need/